### PR TITLE
Fixed stack viewer, loader resetted on each load, buttonbar just a sa…

### PR DESCRIPTION
**Pull request by ddelpiano**
- Fixed the stack viewer, the loader is resetted on each load.
- Buttonbar just a double check for entity not undefined to avoid the UI to crash.
- Menu component wrapped in React.Fragment due to new warning for invalid props that, however, I cannot spot after an extensive check. (will open a separate card for this).

### CODE REVIEW

File: js/components/widgets/stackViewer/StackViewerComponent.js [hunk 1]
```diff
@@ -508,8 +510,10 @@ define(function (require) {
         loadType: PIXI.loaders.Resource.LOAD_TYPE.IMAGE,
         xhrType: PIXI.loaders.Resource.XHR_RESPONSE_TYPE.BLOB
       };
+      let loaderResources = imageLoader.resources;
```
Comment (gidili) [inline js/components/widgets/stackViewer/StackViewerComponent.js]: @ddelpiano  Why assign here and not just on line 515?

File: js/components/widgets/stackViewer/StackViewerComponent.js [hunk 2]
```diff
@@ -925,8 +929,10 @@ define(function (require) {
               }
               this.state.images[d].texture = PIXI.Texture.fromImage(image);
               if (!PIXI.loader.resources[image]) {
+                let loaderResources = PIXI.loader.resources;
                 PIXI.loader.reset();
-                PIXI.utils.clearTextureCache()
+                PIXI.loader.resources = loaderResources;
```
Comment (gidili) [inline js/components/widgets/stackViewer/StackViewerComponent.js]: @ddelpiano same weird double assignment?

File: js/components/widgets/stackViewer/StackViewerComponent.js (hunk 1 above)
Comment (ddelpiano) [reply js/components/widgets/stackViewer/StackViewerComponent.js]: on line 515 I already lost the resources loaded due to the reset() call.

File: js/components/widgets/stackViewer/StackViewerComponent.js (hunk 2 above)
Comment (ddelpiano) [reply js/components/widgets/stackViewer/StackViewerComponent.js]: same reason as above.

**Pull request merged**